### ISSUE TITLE
tfsec default version is broken (again)

### ISSUE THREAD

**Issue by marcus-crane** [aquasecurity/tfsec-sarif-action#30]
Hi there,

The combination of #27 and #28 is now returning `https://api.github.com/repos/aquasecurity/tfsec/releaseslatest` which is also not a valid URL.

I think only one of these PRs should have been merged, not both.

For reference, the above is when using the following configuration:

```yaml
[...]
      - name: tfsec
        uses: aquasecurity/tfsec-sarif-action@master
        with:
          sarif_file: tfsec.sarif #Static Analysis Results Interchange Format (SARIF)
        env:
          github_token: ${{ secrets.github_token }}
```

As a side note, I think this repo should run its own action as a check so you can have visibility into its current working state 😉 

Thanks!

**Comment by marcus-crane** [comment]
I think reverting #27 makes more sense. `/releases` will return all releases whereas `/releases/latest` will return a much smaller JSON payload which is much faster as a default.

**Comment by owenrumney** [comment]
I'll have a proper look on Monday

**Comment by owenrumney** [comment]
should be fixed in v0.1.3 - that'll teach me not to review PRs on my phone
